opensrp-reveal/src/main/res/layout/nav_header.xml

```diff
@@ -174,12 +174,34 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/drawer_margin"
         android:layout_marginTop="@dimen/drawer_margin_vertical"
+        android:layout_marginBottom="10dp"
         android:text="@string/logout_text"
         android:textColor="@color/button_blue"
         android:textSize="18sp"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/divider"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/operator_label"
         app:layout_constraintVertical_bias="0.9" />
 
+    <View
+        android:id="@+id/divider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:background="#DFDFDF"
+        app:layout_constraintStart_toStartOf="parent"
+        android:layout_marginStart="@dimen/divider_margin"
+        app:layout_constraintBottom_toTopOf="@id/attribution"
+        android:layout_marginEnd="@dimen/divider_margin"
+        android:layout_marginBottom="10dp" />
+
+    <TextView
+        android:id="@+id/attribution"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textColor="@color/text_gray"
+        android:layout_marginStart="@dimen/drawer_margin"
+        app:layout_constraintStart_toStartOf="parent"
```

Comment:
Add  `app:layout_constraintEnd_toEndOf="parent"` to automatically wrap if not enough space is available